src/test/ui/label/label_misspelled.rs

```diff
@@ -15,6 +11,10 @@ fn main() {
         for_loop;
         //~^ ERROR cannot find value `for_loop` in this scope
     };
+    'LOOP: loop {
```

Comment:
why was this moved?

Reply:
To avoid the "unreachable code" warning (that doesn't trigger in ui tests unless you enable, but do trigger when testing manually).